apps/arweave/src/ar_node.erl

```diff
@@ -181,8 +181,12 @@ get_current_usd_to_ar_rate() ->
 %% the hashes of the recent blocks that can be used in transactions as anchors.
 %% @end
 get_block_anchors() ->
-	[{block_anchors, BlockAnchors}] = ets:lookup(node_state, block_anchors),
```

Comment:
Could we return an atom here (e.g., not_joined) to avoid bugs?

Reply:
Hm, I think, it makes sense to return 403 not_joined from the new endpoint until the node has been initialized, like in most other endpoints?

Reply:
Updated! https://github.com/ArweaveTeam/arweave/pull/581/commits/9e18942770c521b4dcb50e31cc00641a2cdea790